CMakeLists.txt

```diff
@@ -85,6 +87,19 @@ add_compile_options(/WX /Gy
     # note that /Zi generates debug info inside the object file, it's the same as /Z7 for msvc
     "$<$<COMPILE_LANGUAGE:ASM_MASM>:/Zi;/W3;/nologo>")
 
+if(STL_USE_ANALYZE)
+    # warning C6553: The annotation for function 'LCMapStringEx' on _Param_(9)
+    # does not apply to a value type.
+    # There's a bug in the declaration for LCMapStringEx - it applies _In_opt_ to an LPARAM.
+    # LPARAM is a LONG_PTR (intptr_t), and it's invalid to apply _In_opt_ to a non-pointer.
+
+    # warning C28182: Dereferencing NULL pointer. 'Temp_value_#1457' contains the same NULL value as
+    # 'new(1*40, nothrow)' did.
+    # `new` expressions with allocation functions that may return `null` are incorrectly treated as dereferencing
+    # the new'd pointer. This is a bug in C28182.

```

Comment:
Do we have bug numbers for these yet so we could simply `# TRANSITION, OS-XXXXX and VSO-YYYYYY` here?

Reply:
Half-done: I reduced and filed internal VSO-1558277 "Bogus 'warning C28182: Dereferencing NULL pointer' when using nothrow-new with a unique_ptr and custom deleter", and added TRANSITION comments.

Reply:
For the other half, we just need a `# TRANSITION, OS-XXXXXX` here with the bug @strega-nil-ms filed this morning.